linux_os/guide/system/network/network-ipv6/disabling_ipv6/grub2_disable_ipv6/rule.yml

```diff
@@ -0,0 +1,94 @@
+documentation_complete: true
+
+prodtype: fedora,ocp4,ol7,ol8,rhel7,rhel8,rhv4
+
+title: 'Ensure IPv6 is disabled through kernel boot parameter'
+
+description: |-
+    To disable IPv6 protocol support in the Linux kernel,
+    add the argument <tt>ipv6.disable=1</tt> to the default
+    GRUB2 command line for the Linux operating system in
+{{% if product in ["rhel7", "ol7", "rhv4"] %}}
+    <tt>/etc/default/grub</tt>, so that the line looks similar to
+    <pre>GRUB_CMDLINE_LINUX="... ipv6.disable=1 ..."</pre>
+    In case the <tt>GRUB_DISABLE_RECOVERY</tt> is set to true, then the parameter should be added to the <tt>GRUB_CMDLINE_LINUX_DEFAULT</tt> instead.
+{{% else %}}
+    <tt>/boot/grub2/grubenv</tt>, in the manner below:
+    <pre>sudo  grub2-editenv - set "$(grub2-editenv - list | grep kernelopts) ipv6.disable=1"</pre>
+{{% endif %}}
+    
+
+rationale: |-
+    Any unnecessary network stacks, including IPv6, should be disabled to reduce
+    the vulnerability to exploitation.
+
+severity: low
+
+identifiers:
+    cce@rhel7: 82886-3
+    cce@rhel8: 82887-1
+
+references:
+    cis@rhel7: 3.3.3
+    cis@rhel8: "3.6"
+
+ocil_clause: 'IPv6 is not disabled'
+
+ocil: |-
+    {{% if product in ["rhel7", "ol7", "rhv4"] %}}
+    Inspect the form of default GRUB2 command line for the Linux operating system
+    in <tt>/etc/default/grub</tt>. If it includes <tt>ipv6.disable=1</tt>, then IPv6
+    is disabled at boot time.
```

Comment:
`then IPV6 is disabled at boot time.` -> This give the impression that nothing else needs to be checked. I suggest to remove it.

Change sentence to:
`Check if it includes <tt>ipv6.disable=1</tt>.`